src/use/use.py

```diff
@@ -177,11 +180,11 @@ def fail_or_default(default, exception, msg):
     else:
         raise exception(msg)
 
-def apply_aspect(mod:ModuleType, check:callable, pattern:str, decorator:callable):
+def apply_aspect(mod:ModuleType, check:callable, pattern:str, decorator:Callable[...]):
```

Comment:
nice catch, thanks

Reply:
Actually I think thus was one of the first few warnings when I turned on type checking, that's why I got to it :)